packages/vertexai/src/api.ts

```diff
@@ -42,16 +42,13 @@ declare module '@firebase/component' {
  *
  * @param app - The {@link @firebase/app#FirebaseApp} to use.
  */
-export function getVertex(
-  app: FirebaseApp = getApp(),
-  options?: VertexOptions
-): Vertex {
+export function getVertex(app: FirebaseApp = getApp()): Vertex {
```

Comment:
should this be getVertexAI now?

Reply:
Will fix this in the merge with the other PR.